thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/notification/formatter/channels/JiraContentFormatter.java

```diff
@@ -48,6 +47,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.pinot.thirdeye.notification.commons.ThirdEyeJiraClient.*;
```

Comment:
Can we include the classes explicitly? If there are not too many of them, it is better to include explicitly.

Reply:
I double checked. There is actually more than 5 constants we depend on.